### PR TITLE
Turkish translation chapter 0

### PR DESCRIPTION
**Pull request by akkasayaz**
English version was really smooth and I wanted the same for the Turkish version
#41

### CODE REVIEW

File: .idea/.idea.course-1.dir/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿# Default ignored files
```
Comment (lewtun) [inline .idea/.idea.course-1.dir/.idea/.gitignore:1]: Would you mind removing this file from the PR and adding an `.idea/` to the main `.gitignore`?

**Review by lewtun — APPROVED** [review]
Thanks for this great contribution @akkasayaz ! I've left a small comment about removing a file, but otherwise this looks great :)

File: .idea/.idea.course-1.dir/.idea/.gitignore (hunk 1 above)
Comment (akkasayaz) [reply .idea/.idea.course-1.dir/.idea/.gitignore]: I did it can you check please?

Comment (lewtun) [reply .idea/.idea.course-1.dir/.idea/.gitignore]: Yep, this looks great - thanks!

File: chapters/tr/_toctree.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- title: 0. Setup
```
Comment (lewtun) [inline chapters/tr/_toctree.yml:1]: Feel free to translate this title too if you want :)

File: .github/workflows/build_documentation.yml [hunk 3]
```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: en es th
+      languages: en es th tr
```
Comment (lewtun) [inline .github/workflows/build_documentation.yml:17]: Just FYI I added Turkish to the doc build so we can check it renders on the website :)

**Pull request merged**